File: solutions/beecrowd/3342/3342.py
import sys

for line in sys.stdin:
    n = int(line)

    black_squares = int(n ** 2 / 2)
    white_squares = black_squares
    if not n % 2 == 0:
        white_squares += 1

    print(f"{white_squares} casas brancas e {black_squares} casas pretas")

File: solutions/beecrowd/3342/3342.c
#include <stdint.h>
#include <stdio.h>

int main() {
    int16_t n;
    while (scanf("%d", &n) != EOF) {
        int16_t black_squares = n * n / 2;
        int16_t white_squares = black_squares;

        if (n % 2 != 0) {
            white_squares++;
        }

        printf(
            "%d casas brancas e %d casas pretas\n",
            white_squares,
            black_squares
        );
    }

    return 0;
}
